tray/src/com/dvt/main/creational/abstract_factory/factory/StampingEquipmentFactory.java

```diff
@@ -0,0 +1,13 @@
+<<<<<<< HEAD
+package com.dvt.creational.abstract_factory.factory;
+
+import com.dvt.creational.abstract_factory.Part;
+=======
+package creational.abstract_factory.factory;
+
+import creational.abstract_factory.Part;
+>>>>>>> main
```

Comment:
Please fix merge conflict.
```suggestion
<<<<<<< HEAD
package com.dvt.creational.abstract_factory.factory;

import com.dvt.creational.abstract_factory.Part;
=======
package creational.abstract_factory.factory;

import creational.abstract_factory.Part;
>>>>>>> main
```